### ISSUE TITLE
Protocol between client/server

### ISSUE THREAD

**Issue by JuneToJuly** [Luc-Olsthoorn/TrueShot#8]
We could use a byte array for transferring metadata and be really efficient, but let's just use UTF characters.

**How does this look?**
```
sendToClient = "scenario number"
```
Where scenario number is the scenario selected.

This initiates the start of a scenario, the server enters scenario state where it waits for cordinates from client.

You can sample the rotation every second and then send that over with the following format:
```
cordinates = "x;y;z;"
```
Where x, y, z are the cordinates.

Server processes the sound with these cordinates and sends byte array back to you.

Any suggestions on how you would want to do it @Luc-Olsthoorn? I just threw this out there.

**Comment by Luc-Olsthoorn** [comment]
This looks pretty good! I think that in my mind we would have a continuous stream of xyz being sent to the server, and the server decides what (if any sound) is returned. That way we could generate everything on the server side, and have multiple people listen to the same scenario at once

**Comment by Luc-Olsthoorn** [comment]
True, they are coding wise almost identical, so we can just see them performance wise
